kfs-core/src/main/java/edu/arizona/kfs/tax/TaxConstants.java

```diff
@@ -0,0 +1,79 @@
+package edu.arizona.kfs.tax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaxConstants {
+
+
+    public static final String TAX_USER_ROLE = "UA 1099 Tax Specialist";
+    public static final String NMSPC_CD = "KFS-TAX";
```

Comment:
Why was this changed from 'KAULI-TAX' to 'KFS-TAX'? Note: This also is reflected in the spring-tax.xml in PR 216